pkgs/applications/misc/nwg-displays/default.nix

```diff
@@ -0,0 +1,58 @@
+{ lib
+, fetchFromGitHub
+, wrapGAppsHook
+, atk
+, gdk-pixbuf
+, gobject-introspection
+, gtk-layer-shell
+, gtk3
+, pango
+, python310Packages
+, python3Packages
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "nwg-displays";
+  version = "0.3.7";
+  doCheck = false;
+  dontWrapGApps = true;
+
+  src = fetchFromGitHub {
+    owner = "nwg-piotr";
+    repo = "${pname}";
+    rev = "v${version}";
+    sha256 = "sha256-Y405ZeOSpc1aPKEzFdvlgJgpGAi9HUR+Hvx63uYdp88=";
+  };
+
+  nativeBuildInputs = [
+    wrapGAppsHook
+    gobject-introspection
+  ];
+
+  buildInputs = [
+    gtk3
+  ];
+
+  propagatedBuildInputs = [
+    pango
+    gtk-layer-shell
+    gdk-pixbuf
+    atk
+    python310Packages.i3ipc
+    python310Packages.pygobject3
+    python310Packages.gst-python
+  ];
+
+  preFixup = ''
+    makeWrapperArgs+=("''${gappsWrapperArgs[@]}");
+  '';
+
```

Comment:
```suggestion

  # <insert here a justification why are you disabling check>
  doCheck = false;

```

Reply:
I get when i remove it. I just copied that flag from their old version on their flake.nix in their repo.

```
/nix/store/w453w4jw22kf20x0yv8dg08p4gima8mv-python3.10-setuptools-67.4.0/lib/python3.10/site-packages/setuptools/command/build_py.py:202: SetuptoolsDeprecationWarning:     Installing 'nwg_displays.langs' as data is deprecated, please list it in `packages`.
    !!


    ############################
    # Package would be ignored #
    ############################
    Python recognizes 'nwg_displays.langs' as an importable package,
    but it is not listed in the `packages` configuration of setuptools.

    'nwg_displays.langs' has been automatically added to the distribution only
    because it may contain data files, but this behavior is likely to change
    in future versions of setuptools (and therefore is considered deprecated).

    Please make sure that 'nwg_displays.langs' is included as a package by using
    the `packages` configuration field or the proper discovery methods
    (for example by using `find_namespace_packages(...)`/`find_namespace:`
    instead of `find_packages(...)`/`find:`).

    You can read more about "package discovery" and "data files" on setuptools
    documentation page.


!!

  check.warn(importable)
/nix/store/w453w4jw22kf20x0yv8dg08p4gima8mv-python3.10-setuptools-67.4.0/lib/python3.10/site-packages/setuptools/command/build_py.py:202: SetuptoolsDeprecationWarning:     Installing 'nwg_displays.resources' as data is deprecated, please list it in `packages`.
    !!


    ############################
    # Package would be ignored #
    ############################
    Python recognizes 'nwg_displays.resources' as an importable package,
    but it is not listed in the `packages` configuration of setuptools.

    'nwg_displays.resources' has been automatically added to the distribution only
    because it may contain data files, but this behavior is likely to change
    in future versions of setuptools (and therefore is considered deprecated).

    Please make sure that 'nwg_displays.resources' is included as a package by using
    the `packages` configuration field or the proper discovery methods
    (for example by using `find_namespace_packages(...)`/`find_namespace:`
    instead of `find_packages(...)`/`find:`).

    You can read more about "package discovery" and "data files" on setuptools
    documentation page.


!!

  check.warn(importable)
```